### DISCUSSION TITLE
How to configure Vulkan properly in CloudRendering Mode?

### DISCUSSION THREAD

**Discussion by chengliu-LR** [allenai/ai2thor#1130] [Q&A]
Dear developers,

when I try to run ai2thor in CloudRendering Mode on a Ubuntu 20.04 cluster, I ran into this problem:
`Exception: Unity process has exited - check ~/.config/unity3d/Allen\ Institute\ for\ Artificial\ Intelligence/AI2-THOR/Player.log for errors. Confirm that Vulkan is properly configured on this system using vulkaninfo from the vulkan-utils package. returncode=-11`.
The cluster has a Nvidia GTX1080Ti GPU, which shoud be compatable with Vulkan.
Is there any insights on this error?

Thank you for your help!

**Comment by LarsLiden** [comment]
Anyone get past this?
